tests_v2/test_cli_new_workspace_command_queue.py

```
@@ -89,34 +112,81 @@ def main() -> int:
+            _must(bool(created_layout_ws_id), f"Missing layout workspace id in output: {output!r}")
+            _must(c.current_workspace() == baseline_ws_id, "new-workspace --layout should preserve selected workspace")
+
+            _wait_for_marker(layout_left_marker, layout_left_token)
+            _wait_for_marker(layout_right_marker, layout_right_token)
+            _must(c.current_workspace() == baseline_ws_id, "Layout command execution should not switch selected workspace")
         finally:
+            if created_layout_ws_id:
+                try:
+                    c.close_workspace(created_layout_ws_id)
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)